src/dbspro/cli/run.py

```diff
@@ -78,8 +79,7 @@ def add_arguments(parser):
     parser.add_argument('abcs', default=None, type=str, metavar='<TSV-FILE>',
                          help="Path to tsv file containing the name and sequence of ABCs. "
                               "DEFAULT: Use file ABC-sequences.tsv in construct-info.")
-    parser.add_argument('targets', nargs='*', metavar='<TARGETS>',
-                        default=['umi-counts.txt', 'umi-density-plot.png', 'read-density-plot.png'],
+    parser.add_argument('targets', nargs='*', metavar='<TARGETS>', default=None,
```

Comment:
`default=None` could be omitted.

Reply:
Is `None` always the default (by default) in argparse arguments?

Reply:
[Yes](https://docs.python.org/3/library/argparse.html#default).